### PR TITLE
Style login page

### PR DESCRIPTION
**Pull request by kenzysoror**
## Notion ticket link

[Login Page](https://www.notion.so/uwblueprintexecs/Login-Page-46b6b3b32e8b41ac821a93fa91cdc9d8)

## Implementation description
- Add Sistema (colour) logo and create `SistemaLogoColour` component
- Add "Only sign in with a Sistema email." text to `index.tsx`
- Style `SignInButton` component
- Remove `SignOutButton` and `session` from `index.tsx`

## Steps to test

1. Visit http://localhost:3000/
2. Confirm "Sign in with Google" button functionality
3. Use Developer Tools to experiment with UI on different screen layouts

Desktop Browser UI:
Screenshot 2024-10-08 221117

(Dev Tool) Tablet UI:
Screenshot 2024-10-08 222420

(Dev Tool) iPhone SE UI:
Screenshot 2024-10-08 222501

## What should reviewers focus on?
- UI alignment with mock
screenshot_2024-10-01_at_9 21 19___pm

- UI transferability (different screen sizes, orientations, etc.)
- General code style/best practices

## Checklist

- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
-import { useSession } from 'next-auth/react';
+import { SistemaLogoColour } from '../components/SistemaLogoColour';
+import { Box, Flex, Text } from '@chakra-ui/react';
 
 export default function Index() {
-  const session = useSession();
   return (
-    <>
+    <Flex
+      direction="column" // stack items vertically
```
Comment (ChinemeremChigbo) [inline src/pages/index.tsx:8]: I would remove these comments, they don't add much value.

File: src/components/SistemaLogoColour.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { Image } from '@chakra-ui/react';
+
+export function SistemaLogoColour() {
+  return (
+    <Image src="/images/sistema_logo_colour.png" alt="Sistema Logo Colour" />
```
Comment (ChinemeremChigbo) [inline src/components/SistemaLogoColour.tsx:5]: This image is not well high quality / well cut out lmaooooo. I think the one I have attached should work better.
sistema_logo_colour

**Review by ChinemeremChigbo — CHANGES_REQUESTED** [review]

**Comment by RahulTandon1** [comment]
Hey! I don't think I have much to add to the review aside from what Chinemerem wrote.
If you agree with all the changes he's suggested and you plan on making changes to the code, could you please ping me once you're done with the changes so I can take another look?

File: src/pages/index.tsx (hunk 1 above)
Comment (kenzysoror) [reply src/pages/index.tsx]: Sounds good!

File: src/components/SistemaLogoColour.tsx (hunk 2 above)
Comment (kenzysoror) [reply src/components/SistemaLogoColour.tsx:5]: Thank you for attaching this!

File: src/components/SignInButton.tsx [hunk 3]
```diff
@@ -1,11 +1,23 @@
...
-    <button onClick={() => signIn('google', { callbackUrl: `/homepage` })}>
-      Sign In Google
-    </button>
+    <Button
+      variant="outline"
+      borderColor="blue.400"
+      colorScheme="gray"
+      width="275px"
+      height="45px"
+      color="black"
```
Comment (ChinemeremChigbo) [inline src/components/SignInButton.tsx:15]:
```suggestion
      color="text"
```
**Before:**
image

**After:**
image

File: src/components/SignInButton.tsx [hunk 4]
```diff
@@ -1,11 +1,23 @@
 // importing from next-auth/react instead of our root's auth.ts because of Client & Server component issues.
 // https://authjs.dev/getting-started/session-management/login
 import { signIn } from 'next-auth/react';
+import { Button } from '@chakra-ui/react';
+import { FcGoogle } from 'react-icons/fc';
 
 export function SignInButton() {
```
Comment (ChinemeremChigbo) [inline src/components/SignInButton.tsx:7]:
```suggestion
export function SignInButton(): JSX.Element {
```

File: src/pages/index.tsx [hunk 5]
```diff
@@ -1,18 +1,19 @@
 import { SignInButton } from '../components/SignInButton';
-import { SignOutButton } from '../components/SignOutButton';
-import { useSession } from 'next-auth/react';
+import { SistemaLogoColour } from '../components/SistemaLogoColour';
+import { Box, Flex, Text } from '@chakra-ui/react';
 
 export default function Index() {
```
Comment (ChinemeremChigbo) [inline src/pages/index.tsx:5]:
```suggestion
export default function Index(): JSX.Element {
```

**Review by ChinemeremChigbo — APPROVED** [review]
Please make the suggested changes prior to merging. Also, after you successfully address a comment you should mark it as resolved.

**Review by RahulTandon1 — APPROVED** [review]
Sorry for the late review.
Thank you for the amazing work!!

**Pull request merged**